demo/index.html

```diff
@@ -163,13 +166,16 @@ <h2 id="current_visitors" class="data">...</h2>
       </div>
 
       <section id="explanation">
-        <h3>About this Data</h3>
+        <h3>About this Site</h3>
         <p>
-          This data provides a window into how people are interacting with the government online. The information comes from a unified Google Analytics account for U.S. federal government agencies known as the <a href="https://www.digitalgov.gov/services/dap/">Digital Analytics Program</a>. This program helps government agenices understand how people find, access and use governmnet services online. The program does not collect any personal information from individual users.
+          This data provides a window into how people are interacting with the government online. The data comes from a unified Google Analytics account for U.S. federal government agencies known as the <a href="https://www.digitalgov.gov/services/dap/">Digital Analytics Program</a>. This program helps government agenices understand how people find, access, and use government services online. The program does not collect any personal information about individual users.
         </p>
 
         <p>
-          Not every government website is represented in this data. Currently, the Digital Analytics Program collects web traffic from almost 300 executive branch government domains out of <a href="https://catalog.data.gov/dataset/gov-domains-api">around 1,350 domains</a> total. This data is not a representative sample of all web traffic to government websites because it only includes agencies that are participating in the program.
+          Not every government website is represented in this data. Currently, the Digital Analytics Program collects web traffic from almost 300 executive branch government domains out of <a href="https://catalog.data.gov/dataset/gov-domains-api">about 1,350 domains</a> total. This data is not a representative sample of all web traffic to government domains.
+        </p>
+
+        <p><strong>Top 20 data:</strong> "Now" data includes traffic to a specific, single page, whereas "7 Days" and "30 Days" data includes traffic to a domain AND all sub-pages within that domain. This is how the Google Analytics API releases the data.
```

Comment:
> This is how the Google Analytics API releases the data.

Personally, I think the section is equally descriptive without this. I'm also concerned about this baiting developers to go nitpick us on what the API docs say.